Shared/Game.Shared.External/Providers/Ai/Llama/LlamaAiProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Text.Json;
+using Game.Shared.External.Providers.Ai.Llama.Internal;
+using Microsoft.Extensions.Options;
+
+namespace Game.Shared.External.Providers.Ai.Llama
+{
+    /// <summary>
+    /// Provides access to the Llama AI model.
+    /// </summary>
+    public class LlamaAiProvider : IAiProvider
+    {
+        /// <summary>
+        /// The client used to communicate with the AI provider.
+        /// </summary>
+        private readonly HttpClient _client;
+
+        /// <summary>
+        /// Constructs a new Llama provider.
+        /// </summary>
+        public LlamaAiProvider(IOptions<AiEndpointConfig> opts)
+        {
+            _client = new HttpClient { BaseAddress = new(opts.Value.EndpointUrl) };
```

Comment:
To pewnie trzeba będzie zmienić (być może w przyszłości, nie teraz), bo tego typu tworzenie jest średnio fajne, lepiej tutaj bedzie wstrzykiwać HttpClientFactory (see https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests)

Reply:
zrobię to w jakimś follow-up PRze najwyżej